library/testing/TestSDKWindowAuto.cxx

```diff
@@ -8,7 +8,7 @@
 int TestSDKWindowAuto(int argc, char* argv[])
 {
   f3d::log::setVerboseLevel(f3d::log::VerboseLevel::DEBUG);
-  f3d::engine eng = f3d::engine::create();
+  f3d::engine eng = f3d::engine::create(true);
```

Comment:
any specific reason why you are chaging this test ?

Reply:
Because EGL won't work if not offscreen (happens when X is not running).

Reply:
indeed